packages/transaction-controller/src/TransactionControllerIntegration.test.ts

```diff
@@ -797,6 +797,9 @@ describe('TransactionController Integration', () => {
             isMultichainEnabled: true,
             getPermittedAccounts: async () => [ACCOUNT_MOCK],
             getSelectedAddress: () => ACCOUNT_MOCK,
+            pendingTransactions: {
```

Comment:
I hope this is a quick fix for the sporadic unit test failures.

Depending on the test duration, the pending transaction tracker would automatically resubmit some of the transactions which the mocks weren't setup for.